channel/adjudicator.go

```diff
@@ -78,6 +78,13 @@ type (
 		Secondary bool  // Optimized secondary call protocol
 	}
 
+	// SignedState represents a signed channel state including parameters.
+	SignedState struct {
```

Comment:
Hm, would that have any benefits here? I think we use type `Transaction` in a different context and it may be confusing to reuse it here.

Reply:
The idea was that a `State`, together with the sigs, becomes a `Transaction` because it progresses the channel. But the naming wasn't very good apparently (and it doesn't contain `Params`).

Now looking at it, it would probably be cleaner to also change the `AdjudicatorReq` to contain, or even embed, a `SignedState` and then remove the `Params` and `Tx` fields. Because otherwise the ledger channel's `SignedState` is scrambled from fields of the `AdjudicatorReq` whereas the sub-channels are directly passed as `SignedState`s. You could then also add a `SignedState()` getter to the channel machine.

Reply:
OK, I suggest we do that as a follow up issue. Would you like to create a corresponding issue?

Reply:
opened #122